attributes/default.rb

```diff
@@ -24,19 +27,99 @@
 default['pgpool']['pgconf']['child_max_connections'] = 0
 default['pgpool']['pgconf']['connection_life_time'] = 0
 default['pgpool']['pgconf']['client_idle_limit'] = 0
-default['pgpool']['pgconf']['enable_pool_hba'] = false
+default['pgpool']['pgconf']['enable_pool_hba'] = false # Might want to override
+# this if the use case includes authentication by Pgpool (recommended).
 default['pgpool']['pgconf']['logdir'] = '/var/log/pgpool'
 default['pgpool']['pgconf']['connect_timeout'] = 10000
-default['pgpool']['pgconf']['backend_hostname0'] = ''
+default['pgpool']['pgconf']['backend_hostname0'] = '' # Should override
+default['pgpool']['pgconf']['backend_hostname1'] = '' # Should override
 default['pgpool']['pgconf']['backend_port0'] = 5432
-default['pgpool']['pgconf']['ssl'] = false
-default['pgpool']['pgconf']['ssl_key'] = ''
-default['pgpool']['pgconf']['ssl_cert'] = ''
-default['pgpool']['pgconf']['ssl_ca_cert'] = ''
-default['pgpool']['pgconf']['ssl_ca_cert_dir'] = ''
+default['pgpool']['pgconf']['backend_port1'] = 5432
+default['pgpool']['pgconf']['ssl'] = false # Should override if SSL is desired.
+default['pgpool']['pgconf']['ssl_key'] = '' # Should override
+default['pgpool']['pgconf']['ssl_cert'] = '' # Should override
+default['pgpool']['pgconf']['ssl_ca_cert'] = '' # Should override
+default['pgpool']['pgconf']['ssl_ca_cert_dir'] = '' # Should override
 default['pgpool']['pgconf']['replication_mode'] = false
-default['pgpool']['pgconf']['load_balance_mode'] = false
-
+default['pgpool']['pgconf']['load_balance_mode'] = true
```

Comment:
I agree if there is no other reason for us to use pgpool other than as a LB and read-only/read-write connection handler, which I think is the case.